crates/sui-graphql-rpc/src/types/balance.rs

```diff
@@ -12,6 +14,15 @@ pub(crate) struct Balance {
     pub(crate) total_balance: BigInt,
 }
 
+impl From<sui_json_rpc_types::Balance> for Balance {
+    fn from(balance: sui_json_rpc_types::Balance) -> Self {
+        Self {
+            coin_object_count: balance.coin_object_count as u64,
+            total_balance: BigInt::from_str(&format!("{}", balance.total_balance)).unwrap(),
```

Comment:
We should introduce `From` impls for the various integer types for `BigInt`.

Also note that it's not great that the JSON RPC type uses `usize` here on a type that came across the wire -- it leaves the width of the type ambiguous (e.g. if you build this on a 32bit system -- unlikely though that is -- it could panic).  No action needed on our side, but I wanted to flag this as a pattern that we should not emulate.